mpsl/include/mpsl_fem_config_common.h

```diff
@@ -34,6 +34,12 @@ typedef struct
     uint8_t gpiote_ch_id; /**< The GPIOTE channel used for toggling this pin. */
 } mpsl_fem_gpiote_pin_config_t;
 
+/** @brief Informs protocol stacks using MPSL that fix for errata FTPAN-254 shall be applied.
+ *
+ * @note This function shall be called before initialization of any protocol stack.
+ */
+void mpsl_errata_254_apply(void);
```

Comment:
I'm confused about this part, is this just a set function? Or does it apply the fix?

This function and `mpsl_fem_errata_254_shall_apply` does not look like set and get functions as the changelog entry is saying.

Reply:
From a point of view of an application, `mpsl_errata_254_apply` "applies" the fix. However it is just a "setter" function as the whole mechanism is abstracted and shall be implemented by a protocol stack, which queries whether the fix should be applied using the "getter" `mpsl_fem_errata_254_shall_apply`.

Reply:
Shall I change the entry to;

Adding API to enable application of FTPAN-254. The function provided does not apply the fix upon return, but instead it flags the intention from the stack of applying the fix. The fix will be then applied when/where necessary.